data/en/cfheader.json

```diff
@@ -13,7 +13,7 @@
 
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-g-h/cfheader.html"},
+		"coldfusion": {"minimum_version":"", "notes":"Script-based cfheader() added in CF11", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-g-h/cfheader.html"},
```

Comment:
You could add a plus sign, so it could be transformed into a label properly: `CF11+` and put it at the beginning of notes.